src/pages/Landing/index.tsx

```diff
@@ -135,15 +150,16 @@ export default () => {
             id='isCool'
             onChange={(value) => setNewUser({ ...newUser, isCool: value })}
             isChecked={newUser.isCool}
+            className={darkmode ? 'dark-mode' : ''}
```

Comment:
'dark-mode' isn't an actual class.    There are likely a few ways to solve this, but what i'd do is:
1.  create a new class above within useStyles for the darkCheckbox, set the color to white.  This won't actually affect the label, but will apply the 'darkCheckbox' style to the checkbox.  
2. here, you'll want to set darkmode as its set elsewhere:  `className={darkmode ? classes.darkCheck : ''}`
3. To make the Label for the checkbox also have white text, i think we need to target it expliclty.  Take a look at the jss docs:  https://cssinjs.org/react-jss/?v=v10.3.0   notice how it targets the span within mybutton:
```
const useStyles = createUseStyles({
  myButton: {
    color: 'green',
    margin: {
      // jss-plugin-expand gives more readable syntax
      top: 5, // jss-plugin-default-unit makes this 5px
      right: 0,
      bottom: 0,
      left: '1rem'
    },
    '& span': {
      // jss-plugin-nested applies this to a child span
      fontWeight: 'bold' // jss-plugin-camel-case turns this into 'font-weight'
    }
  },
  myLabel: {
    fontStyle: 'italic'
  }
})
```

you'll want to do something similar to target the "<label>" element, with a "color: white !important"

Reply:
To help, here's what the rendered html of the component is.  

```
<div class="pf-c-check darkCheck">
   <input id="isCool" class="pf-c-check__input" type="checkbox" aria-invalid="false" data-ouia-component-type="PF4/Checkbox" data-ouia-safe="true" data-ouia-component-id="OUIA-Generated-Checkbox-2">
   <label class="pf-c-check__label" for="isCool">
          Is this person cool? 
    </label>
</div>
```
so by targeting the inner label, we can force its color to be white too